pyodide-test-runner/pyodide_test_runner/tests/test_decorator.py

```diff
@@ -1,83 +1,53 @@
-import asyncio
```

Comment:
Could you please comment why we need to refactor this file? Maybe Hood should review this.

Reply:
I refactored this to remove all `import pyodide` statements. I wanted to make `pyodide-test-runner` testable without accessing Pyodide source tree.

What I mostly have done is replacing the selenium mock class with a real selenium class which, I guess, does not harm test coverage.